templates/gce.tf.erb

```diff
@@ -1,12 +1,6 @@
-provider "google" {
-  region      = "<%= ENV['GCE_REGION'] %>"
-  project     = "govuk-publishing-dns"
-  credentials = "${file("<%= ENV['GCE_CREDENTIALS'] %>")}"
-}
-
 <% records['records'].each_with_index do | record, index | %>
 resource "google_dns_record_set" "<%= record['subdomain'].gsub('.', '_').gsub('@', 'AT') %>_<%= index %>" {
-  managed_zone = "<%= ENV['GCE_ZONE_ID'] %>"
+  managed_zone = "<%= ENV['GOOGLE_MANAGED_ZONE'] %>"
```

Comment:
The changed environment variables should be reflected in the Rakefiles `REQUIRED_ENV_VARS` setting for environment validation